e2e/framework/config.go

```diff
@@ -154,3 +156,11 @@ func (t *TestServerConfig) SetShowsLog(f bool) {
 func (t *TestServerConfig) SetEpochSize(epochSize uint64) {
 	t.EpochSize = epochSize
 }
+
+func (t *TestServerConfig) SetMinValidatorCount(val uint32) {
+	t.MinValidatorCount = val
+}
+
+func (t *TestServerConfig) SetMaxValidatorCount(val uint32) {
+	t.MaxValidatorCount = val
+}
```

Comment:
Please add the method descriptions

Reply:
fixed a91c866e0f6d742a0cc4af63f07196b4703ec96a